pkg/printers/name.go

```diff
@@ -65,15 +65,15 @@ func (p *NamePrinter) PrintObj(obj runtime.Object, w io.Writer) error {
 	groupVersionKind := obj.GetObjectKind().GroupVersionKind()
 	if len(groupVersionKind.Kind) > 0 {
 		if mappings, err := p.Mapper.RESTMappings(groupVersionKind.GroupKind(), groupVersionKind.Version); err == nil && len(mappings) > 0 {
-			fmt.Fprintf(w, "%s/%s\n", mappings[0].Resource, name)
+			fmt.Fprintf(w, "%s.%s/%s\n", mappings[0].GroupVersionKind.Kind, mappings[0].GroupVersionKind.Group, name)
```

Comment:
Alright, spoke with @smarterclayton and he's nervous about changing case.  tolowering the output will keep the output looking very similar and only look weird for the edges.  Can the matching still be done on a case-insenstive kind?

Reply:
> Can the matching still be done on a case-insenstive kind?

Not with the current changes. It would work if I added the kind as if it was the resource's name here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/discovery/restmapper.go#L110

```go
kind := gv.WithResource(strings.ToLower(resource.Kind))
versionMapper.AddSpecific(gv.WithKind(resource.Kind), kind, kind, scope)
```

which would map a case-insensitive `Kind` to the gvk

Reply:
noooo... just the `kind := gv.WithResource(strings.ToLower(resource.Kind))` expression should give us pause.

Reply:
@liggitt I'm addressing case-insensitive `Kind` matching here: https://github.com/kubernetes/kubernetes/pull/59353/commits/9903dc0b049bece9932d82a9044b00f7625a1778#diff-1d31ff0362cb7ce1739dd9306ba5ecd1

ptal